### ISSUE TITLE
Scarab Bunny Flag is visible!

### ISSUE THREAD

**Issue by Krilliac** [antisocial89/RealTBC_Issue_tracker#97]
http://prnt.sc/b195np

ExtraFlags in the DB needs to be set to 128

**Comment by antisocial89** [comment]
Is this for "Boiling blood" quest? If so, the scarabs don't spawn at all at the pots when you use the blood on them. Also, turning the quest in doesn't spawn any scarabs.

**Comment by Krilliac** [comment]
Nope, unrelated with the quest.. the npc is visible and should be invisible.

**Comment by Saben65** [comment]
```SQL
UPDATE creature_template SET ExtraFlags=ExtraFlags | 128 WHERE entry=21391;
```

**Comment by Saben65** [comment]
NPC is completely gone now, even with GM on. >.<

**Comment by antisocial89** [comment]
I still don't know what the Scarab bunnies are for, is it a trigger for the Scarabs that spawn when turning in the boiling blood quest? I could have swore that's what they would be for however I can't see these with gm on or off even before applying the supplied update.

**Comment by Saben65** [comment]
When you turn in the quest he uses the potion on the chemistry set thing and the trigger is supposed to spawn more scarab I believe.
